### PR TITLE
Replace schedule with text about the upcoming schedule

### PR DESCRIPTION
**Pull request by adrianmoisey**
Fixes https://github.com/scaleconf/scaleconf.github.com/issues/157

### CODE REVIEW

**Comment by vhata** [comment]
Nah, we aren't announcing the Business Day any more, we've already done it.

Something like:

ScaleConf runs over <strong>three days</strong>.  We start with one day of fireside chats focussing on <strong>the business of scale</strong>, while the other two days are presentations from local and international experts on the <strong>technologies, problems and solutions</strong> that they have encountered in building sites and services that can handle the rigours of today's internet.

The Business of Scale day focuses on the issues leaders and their teams face when <strong>growing a technology-driven organisation</strong>. You'll get to hear from those who've faced the challenges of building engineering teams, growing healthy customer acquisition strategies, maintaining inspiring working environments, balancing customer demands and technical debt, dealing with exit opportunities and more.

The second and third days follow our usual formula of <strong>12 x 45 minute talks</strong> plus regular breaks to keep the legs from falling asleep and awesome lunches to feed the mind and body.

There will be <strong>three social events</strong> starting with the Wednesday evening with a sponsored drinks event in the city. Then similar events on the Thursday and Friday nights. These are a chance to socialise and network with the speakers, organisers, and fellow conference attendees.

**Comment by adrianmoisey** [comment]
That looks better. Updated :bell:

File: schedule/index.html [hunk 1]
```diff
@@ -82,3 +83,38 @@
...
     </div>
 </div>
+-->
+<div class="row white-background">
+  <div class="col-xs-12 col-md-12">
+    <p>
+      ScaleConf runs over <strong>three days</strong>. We start with one day of
+      fireside chats focussing on <strong>the business of scale</strong>, while the
+      other two days are presentations from local and international experts on
+      the <strong>technologies, problstrongs and solutions</strong> that they have
```
Comment (vhata) [inline schedule/index.html]: err...  problstrongs

File: schedule/index.html [hunk 2]
```diff
@@ -82,3 +83,38 @@
...
+    </p>
+
+    <p>
+      The second and third days follow our usual formula of <strong>12 x 45 minute
+      talks</strong> plus regular breaks to keep the legs from falling asleep and
+      awesome lunches to feed the mind and body.
+    </p>
+
+    <p>
+      There will be <strong>three social events</strong> starting with the
```
Comment (vhata) [inline schedule/index.html]: s/starting with/starting on/

File: schedule/index.html [hunk 3]
```diff
@@ -82,3 +83,38 @@
...
+      Wednesday evening with a sponsored drinks event in the city. Then similar
+      events on the Thursday and Friday nights. These are a chance to socialise
+      and network with the speakers, organisers, and fellow conference attendees.
```
Comment (vhata) [inline schedule/index.html]: Actually, just:

Each evening we will have a <strong>social event</strong>, to give you a chance to socialise and network with speakers, organisers, and your fellow conference attendees.

**Comment by vhata** [comment]
Sorry, should've proof-read before I pasted.

**Comment by adrianmoisey** [comment]
> Sorry, should've proof-read before I pasted.

It's ok, I shouldn't have actually read it instead of copypasta.

**Comment by adrianmoisey** [comment]
:bell:

**Comment by vhata** [comment]
thumb

**Pull request merged**